src/main/java/edu/ucsb/cs156/happiercows/jobs/InstructorReportJob.java

```diff
@@ -0,0 +1,11 @@
+package edu.ucsb.cs156.happiercows.jobs;
```

Comment:
Need to write a test for this.   

This file has a good example of how to do that.

https://github.com/ucsb-cs156-f22/f22-6pm-happycows/blob/main/src/test/java/edu/ucsb/cs156/happiercows/jobs/MilkTheCowsJobTests.java